test/graphql/mutations.tests.js

```diff
@@ -0,0 +1,516 @@
+const EasyGraphQLTester = require('easygraphql-tester');
+const {
+  MULTIPLE_RESULTS_EVENT,
+  PAGINATED_EVENTS,
+  PAGINATED_BETS,
+  PAGINATED_RESULT_SETS,
+  PAGINATED_WITHDRAWS,
+} = require('./schema-helper');
+const schema = require('../../src/graphql/schema');
+
+const tester = new EasyGraphQLTester(schema);
+
+describe('graphql/mutations', () => {
+  describe('add-pending-event', () => {
+    it('Pass all arguments', async () => {
```

Comment:
nit: write the test name like you would read it as a sentence: `it passes with all arguments`